src/Controls.js

```diff
@@ -74,10 +74,11 @@ var Controls = React.createClass({
         </select>{' '}
         <input ref='start' type='text' />–
         <input ref='stop' type='text' />{' '}
-        <button>Go</button>{' '}
-
-        <button onClick={this.zoomOut}>-</button>{' '}
-        <button onClick={this.zoomIn}>+</button>
+        <button className='btn-submit'>Go</button>{' '}
```

Comment:
Nope, not a bootstrap thing. Was just something I made up/something I usually use on buttons.